ui/src/EventForm.tsx

```diff
@@ -1,43 +1,53 @@
 import React, { useState } from "react";
-import { formatDate, getDateTimeString, padNumberWith0Zero } from "./lib";
+import {
+  formatDate,
+  getDateTimeString,
+  padNumberWith0Zero,
+  timeFormat,
+} from "./lib";
 import { Checkbox, RadioGroup, Radio, Stack } from "@chakra-ui/react";
 import s from "./EventForm.module.css";
 
 interface FormProps {
-  initialStartDate: string;
-  initialEndDate: string;
-  initialStartTime: string;
-  initialEndTime: string;
+  initialStart: string; // ex. "2023-01-25T08:00:00.000Z"
+  initialEnd: string; // ex. "2023-01-25T09:00:00.000Z"
   initialTitle: string;
   initialDescription: string;
   initialAllDay: boolean;
   initialRecurring: boolean;
-  initialRecurrenceEnd: string;
+  initialRecurrenceEnd: string; // ex. "2023-01-25T08:00:00.000Z"
   onFormSubmit: Function;
   isCreate: boolean;
 }
 
 const EventForm = ({
-  initialStartDate,
-  initialEndDate,
-  initialStartTime,
-  initialEndTime,
-  initialTitle,
+  initialStart,
+  initialEnd,
+  initialTitle = "",
   initialDescription = "",
   initialAllDay,
   initialRecurring,
   initialRecurrenceEnd,
   onFormSubmit,
   isCreate,
 }: FormProps) => {
-  const [startDate, setStartDate] = useState<string>(initialStartDate);
-  const [endDate, setEndDate] = useState<string>(initialEndDate);
-  const [startTime, setStartTime] = useState<string>(initialStartTime);
-  const [endTime, setEndTime] = useState<string>(initialEndTime);
+  console.log(initialStart, initialEnd, initialRecurrenceEnd);
+  const [startDate, setStartDate] = useState<string>(
+    formatDate(new Date(initialStart)),
+  );
+  const [endDate, setEndDate] = useState<string>(
+    formatDate(new Date(initialEnd)),
+  );
+  const [startTime, setStartTime] = useState<string>(
+    timeFormat(new Date(initialStart)),
+  );
+
+  const [endTime, setEndTime] = useState<string>(
+    timeFormat(new Date(initialEnd)),
+  );
+
   const [error, setError] = useState<string | null>(null);
   const [title, setTitle] = useState<string>(initialTitle);
-  // `|| ""` prevents warning in tests
```

Comment:
Do we no longer need this?

Reply:
The default behavior was moved to a different line, and it is clear enough that I didn't think a comment was needed.